### DISCUSSION TITLE
AIRAC 2202

### DISCUSSION THREAD

**Discussion by RadarController** [IVAO-XU/EG-Sector-File#71] [General]
Changes for AIRAC 2202: https://nats-uk.ead-it.com/cms-nats/export/sites/default/en/Publications/AIP/EG_Amdt_A_2022_02_en.pdf

MASSIVE change coming this month, any changes MUST be made in the AIRAC2202 branch and not the Master.

- [x] New STARs for EGGW
- [ ] New airspace structure for EGSS/EGGW
- [ ] Establishment of the Argyll, Forth, Moray and Tay CTAs and the extension to the Borders CTA 12 and 13
- [ ] Establishment of Clacton CTA 10, 11 and 12. Daventry CTA 21 amended
- [ ] Establishment of Daventry CTA 25
- [x] Establishment of new points ZAGZO, JUMZI, EDCOX, COCCU, UDDIZ, OXDUF, ILLOC, OFJES, MUCTE, WIQID, OZZOT and MOREZ
- [x] Update to VALBO coordinates
- [ ] Introduction of Flight Plan buffer zones for EG D323 complex
- [x] TRA 003 amended
- [x] HENLOW (MIL) (TRNG AD) - Removed due to aerodrome closure. 
- [x] AD 2.EGCN-5 AD 2 EGCN 5-1 Update to Doncaster RADAR frequencies
- [x] AD 2.EGGW AD 2.2 Aerodrome geographical and administrative data - AD elevation updated.
- [x] AD 2.EGGW AD 2.18 Air traffic services communication facilities - Removal of ESSEX RADAR callsign and 128.750 MHz frequency
- [ ] AD 2.EGSS-4 AD 2 EGSS 4-1 LTMA 2 removed. LTMA 3 and Stansted CTA 3 updated.
- [ ] AD 2.EGHE-2 AD 2 EGHE 2-1 New Taxiway Hotel and hold H. Main taxiway renamed TWY A
- [ ] AD 2.EGPB-3 AD 2 EGPB 3-1 and 3-1 Moray TMZ and Class E CTA 3,4,5 and 16 added.

Any/all help very much appreciated for this huge change.

**Comment by louielister** [comment]
Currently working on EGGW STARs.

**Comment by Bucklerz** [comment]
Split into seperate issues.  @louielister @jakee2603 you will need to link your PRs/Changes to the relevant issues.

**Comment by RadarController** [comment]
Made this a discussion now just in case anyone has any general questions/requests/etc.!

**Comment by RadarController** [comment]
@louielister 

![image](https://user-images.githubusercontent.com/97064113/149828101-67a805ab-e25b-41c6-8419-f665b9093730.png)

**Comment by RadarController** [comment]
And then with the holds in (and less offensive colours!)

![image](https://user-images.githubusercontent.com/97064113/149830230-4754eaf1-ada3-4980-b42b-5ef0be4d47b1.png)
